test/metabase/api/database_test.clj

```diff
@@ -51,7 +51,7 @@
     (mt/object-defaults Database)
     (select-keys db [:created_at :id :details :updated_at :timezone :name])
     {:engine   (u/qualified-name (:engine db))
-     :features (map u/qualified-name (driver.u/features driver))})))
+     :features (map u/qualified-name (driver.u/features driver nil))})))
```

Comment:
I think we should have a guarantee that `database-supports?` is always called with a valid database. Otherwise we have somewhere between 10 and 20 drivers that potentially need to check and make sure `database` isn't nil.

Why don't you just pass `db` here? I see it two lines above this one. You can use `(mt/db)` in other places to get the current test DB for the current driver in other tests